media_kit/lib/src/player/native/player/real.dart

```diff
@@ -706,6 +712,19 @@ class NativePlayer extends PlatformPlayer {
       await waitForPlayerInitialization;
       await waitForVideoControllerInitializationIfAttached;
 
+      // External List<Media>:
+      // ---------------------------------------------
+      final map = SplayTreeMap<double, Media>.from(
```

Comment:
I hope this `move` matches internal libmpv implementation (same is used on web; though passes unit-tests).

Reply:
I didn't know `SplayTreeMap` and the `map[to - 0.5]` trick is very elegant. Bravo!